saga-distribution/src/release/LICENSE

```diff
@@ -353,45 +353,48 @@ For details, see their respective project links.
 
 Apache Commons Logging (http://commons.apache.org/proper/commons-logging/) commons-logging:commons-logging:jar:1.2
 Bean Validation API (http://beanvalidation.org) javax.validation:validation-api:jar:1.1.0.Final
-ClassMate (http://github.com/cowtowncoder/java-classmate) com.fasterxml:classmate:bundle:1.3.3
-com.google.api.grpc:proto-google-common-protos (https://github.com/googleapis/googleapis) com.google.api.grpc:proto-google-common-protos:jar:0.1.9
+ClassMate (http://github.com/cowtowncoder/java-classmate) com.fasterxml:classmate:bundle:1.3.4
+com.google.api.grpc:proto-google-common-protos (https://github.com/googleapis/googleapis) com.google.api.grpc:proto-google-common-protos:jar:1.0.0
 dubbo (https://github.com/apache/incubator-dubbo) com.alibaba:dubbo:jar:2.6.1
-error-prone annotations (http://nexus.sonatype.org/oss-repository-hosting.html/error_prone_parent/error_prone_annotations) com.google.errorprone:error_prone_annotations:jar:2.0.19
+error-prone annotations (http://nexus.sonatype.org/oss-repository-hosting.html/error_prone_parent/error_prone_annotations) com.google.errorprone:error_prone_annotations:jar:2.1.2
 FindBugs-jsr305 (http://findbugs.sourceforge.net/) com.google.code.findbugs:jsr305:jar:3.0.0
 Gson (https://github.com/google/gson/gson) com.google.code.gson:gson:jar:2.7
```

Comment:
Do we use these two same gson version?

Reply:
Yes, both the gson-2.7.jar and gson-2.8.4.jar are included in the release.

Reply:
Updated the LICENSE and add description for the 2 versions of gson jar.